Description: Patch to use debian "reactive_streams" and "rxjava3"
 Fully implemented upstream in tag 6.0.0-pre.20220324.1
Author: Jesse Chan <jc@linux.com>
Forwarded: not-needed
Last-Update: 2022-10-17

--- a/tools/distributions/debian/debian_java.BUILD
+++ b/tools/distributions/debian/debian_java.BUILD
@@ -459,3 +459,16 @@
         "java/junit4.jar",
     ],
 )
+
+# libreactive-streams-java
+java_import(
+    name = "reactive_streams",
+    jars = ["reactive-streams.jar"],
+)
+
+# librx-java
+java_import(
+    name = "rxjava3",
+    jars = ["rxjava.jar"],
+    deps = [":reactive_streams"],
+)
